### PR TITLE
ipmitool v. 1.8.16

### PR DESCRIPTION
**Pull request by rickard-von-essen**

### CODE REVIEW

File: Library/Formula/ipmitool.rb [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+                          "--mandir=#{man}",
+                          "--disable-intf-usb"
     system "make", "install"
   end
 end
```
Comment (mikemcquaid) [inline Library/Formula/ipmitool.rb]: Any chance you could add a `test do` block to this formula (even if it's just doing `-v` or `--help`)? Thanks!

File: Library/Formula/ipmitool.rb [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
 
   depends_on "openssl"
 
   def install
-    # tracking upstream: http://sourceforge.net/p/ipmitool/feature-requests/47/
-    # fix build errors w/ clang
-    inreplace "include/ipmitool/ipmi_user.h", "HAVE_PRAGMA_PACK", "DISABLE_PRAGMA_PACK"
-    # s6_addr16 is not available in Mac OS X
-    inreplace "src/plugins/ipmi_intf.c", "s6_addr16", "s6_addr"
+    inreplace "src/plugins/ipmi_intf.c", "#define _GNU_SOURCE 1", "#define _GNU_SOURCE 1\n#define _DARWIN_C_SOURCE 1"
```
Comment (mikemcquaid) [inline Library/Formula/ipmitool.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

**Comment by rickard-von-essen** [comment]
Submitted the patch upstream and added some info in formula and added a trivial test.

File: Library/Formula/ipmitool.rb [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
     system "./configure", "--disable-debug",
                           "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
-                          "--mandir=#{man}"
+                          "--mandir=#{man}",
+                          "--disable-intf-usb"
     system "make", "install"
   end
+
+  def test
```
Comment (DomT4) [inline Library/Formula/ipmitool.rb]: This should be `test do` rather than `def test`.

**Comment by rickard-von-essen** [comment]
@DomT4 Fixed

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/148720cd68077c6bb31a5b54b0072d9845323706. Thank you for your contribution to Homebrew @rickard-von-essen; we appreciate it! :smiley_cat:

**Pull request closed without merging**